HTMLSerializer.js

```diff
@@ -112,6 +126,30 @@ var HTMLSerializer = class {
   }
 
   /**
+   * Takes an html node of type Node.TEXT_NODE, and add its text content with
+   *     all characters properly escaped to |this.html|.
+   * @param {Node} node The text node.
+   */
+   // TODO(sfine): Take care of attribute value normalization:
+   // https://developers.whatwg.org/the-iframe-element.html#the-iframe-element
+  processText(node) {
```

Comment:
@chrishtr, I thought the same thing but I was unable to find any existing code for this, either on the web or in google3. I think it's just uncommon to try to flatten frametrees using srcdoc. We found the spec link actually describes the algorithm added here which made me feel a little better.